docs/current_docs/quickstart/agent/snippets/go/main.go

```diff
@@ -15,7 +15,7 @@ func (m *CodingAgent) GoProgram(
 	environment := dag.Env().
 		WithToyWorkspaceInput("before", workspace, "these are the tools to complete the task").
 		WithStringInput("assignment", assignment, "this is the assignment, complete it").
-		WithToyWorkspaceOutput("after")
+		WithToyWorkspaceOutput("after", "the ToyWorkspace with the completed assignment")
```

Comment:
We don't use the module name in the input prompt, but we use it in the output prompt. Should both prompts refer to the object in the same way?

Reply:
Not necessary IMO. In the input, it can see the type and it's functions. For the output, it could help to provide guidance